app/src/main/java/com/x8bit/bitwarden/data/autofill/fido2/manager/Fido2CredentialManagerImpl.kt

```diff
@@ -123,6 +125,11 @@ class Fido2CredentialManagerImpl(
             null
         }
 
+    override fun getPasskeyGetCredentialsOptionsOrNull(
+        requestJson: String,
+    ): PasskeyGetCredentialsOptions? =
+        json.decodeFromStringOrNull<PasskeyGetCredentialsOptions>(requestJson)
```

Comment:
I'm curious why we're using `decodeFromStringOrNull` here and `decodeFromString` with custom `catch` blocks in the other methods. Is there a reason for that, or can we just update the others to use `decodeFromStringOrNull` as well for consistency?

Reply:
I saw it pop up as a suggestion. It's doing the same thing we're doing with the custom catches. Since we don't need special handling of the errors, this seems appropriate. We can update the other functions to do the same.